src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Services/Structure/BlockStructureHelpers.cs

```diff
@@ -5,61 +5,30 @@
 #nullable disable
 
 using System.Collections.Immutable;
-using System.Linq;
-using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.CodeAnalysis.Host.Mef;
 using Microsoft.CodeAnalysis.PooledObjects;
 
 namespace Microsoft.CodeAnalysis.Structure
 {
-    internal abstract class BlockStructureServiceWithProviders : BlockStructureService
+    internal static class BlockStructureHelpers
```

Comment:
`BlockStructureServiceWithProviders` was split to move the core structure creation APIs to `BlockStructureHelpers` helper class. All the code shown as deleted in this file is just in a separate file `BlockStructureServiceWithProviders.cs`